frontend/src/components/visual-editor/v2/Diagrams.tsx

```diff
@@ -429,6 +446,32 @@ const Diagrams = () => {
       deleteDialogCtl.closeDialog();
     }
   };
+  const onMove = async (newCategoryId?: string) => {
+    if (!newCategoryId) {
+      return;
+    }
+
+    const ids = moveDialogCtl?.data;
+
+    if (ids?.length && Array.isArray(ids)) {
+      await updateBlocks({ ids, payload: { category: newCategoryId } });
+
+      queryClient.invalidateQueries({
+        predicate: ({ queryKey }) => {
+          const [qType, qEntity] = queryKey;
+
+          return (
+            qType === QueryType.collection &&
+            isSameEntity(qEntity, EntityType.BLOCK)
+          );
```

Comment:
I think we need to not only invalidate the collection but also the items ? Or are they already updated ? (category)